grapher/core/Grapher.tsx

```diff
@@ -423,29 +425,47 @@ export class Grapher
         this.timelineFilter = getTimeDomainFromQueryString(time)
     }
 
-    @computed get table() {
+    @computed private get tablePreTimelineFilter() {
         let table = this.inputTable
         // todo: could make these separate memoized computeds to speed up
         // todo: add cross filtering. 1 dimension at a time.
         table = this.minPopulationFilter
             ? table.filterByPopulation(this.minPopulationFilter)
             : table
-        table = this.filterByTime(table)
+
+        if (
+            this.tab === GrapherTabOption.chart ||
+            this.tab === GrapherTabOption.map
+        ) {
+            const chartComponent = getChartComponent(
```

Comment:
Ah, any idea how to type getChartComponent? I couldn't figure it out earlier. I'll give it another look too.